services/web/server/src/simcore_service_webserver/workspaces/_models.py

```diff
@@ -46,6 +48,14 @@ class WorkspacesFilters(Filters):
         default=False,
         description="Set to true to list trashed, false to list non-trashed (default), None to list all",
     )
+    text: Annotated[
+        str | None, BeforeValidator(empty_str_to_none_pre_validator)
+    ] = Field(
+        default=None,
+        description="Multi column full text search",
+        max_length=100,
+        examples=["My Workspace"],
```

Comment:
to be honest, an example like this does not add much. On the other hand, an example with `%` (or` *`) tokens might be interesting. Make sure to have the examples you have in your tests.